python/tvm/autotvm/measure/measure_methods.py

```diff
@@ -186,6 +189,16 @@ def __init__(self,
                  timeout=10, n_parallel=None,
                  number=4, repeat=3, min_repeat_ms=0, cooldown_interval=0.1,
                  check_correctness=False):
+        static_tune = os.getenv("TVM_STATIC_TUNE_EXPERIMENTAL")
```

Comment:
Can you elaborate on what TVM_STATIC_TUNE_EXPERIMENTAL is being used for? I'm a little weary of inserting additional execution modes in AutoTVM dictated by environmental variables.

(1) we either add additional flags in AutoTVM config
(2) we clean this experimental mode if it's not really necessary for the OpenCL variant of VTA

Reply:
`TVM_STATIC_TUNE_EXPERIMENTAL` is used for static auto tune. I think as you advised, we can split the PR into separate PRs. For static auto tune, the interface is not well nicely done. I think maybe we remove this part first.